outputs/Console.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace GameOfLife\outputs;
+
+use GameOfLife\Field;
+use Ulrichsg\Getopt;
+
+/**
+ * Creates Console-output
```

Comment:
This class does not **create** a console-output, it **is** a console output.
Another good explanation would have been: "Outputs fields to the Console".